RELEASE_NOTES.md

```diff
@@ -10,9 +10,81 @@
+
+#### Changed
+
+* **`ExternalDNS`**: Fixed a bug where `cray-externaldns-manager` could panic if it couldn't connect to PowerDNS on startup
+* **PowerDNS**: Changed `powerdns-manager` SLS error message to debug
+* **PowerDNS**: Fixed bug that caused `powerdns-manager` and externaldns-manager to update the same record
+* **PowerDNS**: Fixed a bug that could cause `powerdns-manager` to intermittently crash when performing a lookup for an existing TSIG key
+* **PowerDNS**: `powerdns-manager` will now retry attempts to add a TSIG key
+* **PowerDNS**: `powerdns-manager` will now create PTR records that are not created by `external-dns`
+* **Unbound**: Changed `cray-dns-unbound` `MaxUnavailable` default to 1 to avoid issues when evicting pods from NCNs
+
+### Management Network
+
+#### Added
+
+* **Documentation**: Added procedure to migrate from the customer access network (CAN) to the customer high-speed network (CHN), allowing user traffic over
+  the HSN instead of the NMN (This is an extension of the `bi-furcated` CAN feature that shipped in CSM 1.2)
+* **Documentation**: Added various troubleshooting guides
+* **Hardware**: Validated the Aruba 8360 (JL705C) switch for the management network
+
+#### Changed
+
+* **CANU**: Fixed bug where UAN VLANs in generated switch configurations were wrong when using the CHN
+* **CANU**: Other various bug fixes (see https://github.com/cray-hpe/canu for a full changelog)
+* **CANU**: Added an ACL on systems with Dell and Mellanox switches to prevent high-speed network (HSN) switches on the Hardware Management Network (HMN)
+  from communicating with the Fabric Manager service on the Node Management Network (NMN) API gateway
+* **CSI**: Fixed bug where CSI could generate bad SLS chassis data
+* **Documentation**: Admins are now asked to updated CANU to the latest version when beginning a CSM upgrade
+* **Documentation**: Management network switch upgrade instructs are now separate from the CSM upgrade procedure
+* **Documentation**: Removed stale reference to Kea `postgres` from troubleshooting documentation
+* **Documentation**: Fixed invalid commands in `generate_switch_configs.md`
+
+### Management Nodes (Ceph, Kubernetes Workers, and Kubernetes Managers)
+
+#### Added
+
+* **ALL**: Initial support for NVME drives (note: there are some limitations in the Dracut modules that handle drive discovery)
+
+#### Changed
+
+* **All**: Updated kernel to `kernel-default-5.3.18-150300.59.87.1`
+* **All**: Various package updates to apply latest security patches
+* **All**: `kdump` is now more reliable and remains functional after IMS image customization
+* **All**: S3 now requires authentication to retrieve Management Node OS images
+* **All**: `metal.no-wipe=1` is now more reliable when running during a net boot
+* **All**: SSH keys must now be injected into images
+* **All**: Time synchronization is now more reliable on initial configuration
+* **All**: Pressure Stall Information (PSI) is now enabled by default
```

Comment:
`s/Stall/Stats/g`

Could hyper-link to this heading for more information: https://github.com/Cray-HPE/docs-csm/tree/main/background#psi

Reply:
The ticket refers to them as stats, but they're known as Pressure Stall Information in the kernel world.  https://docs.kernel.org/accounting/psi.html